### PR TITLE
Add Andrew Hou

### PR DESCRIPTION
**Pull request by andrewfhou**
### Submission Checklist

- [x] Your pull request targets the `master` branch of the repository.
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] You have read the [Contributing guidelines](https://github.com/jboss-outreach/gci/blob/master/CONTRIBUTING.md) and your changes follow them.

#### Type of Change

- [ ] Bug fix
- [x] New implementation

### PR Description

Add Andrew Hou's profile and image

Issue #[Add issue number here.]

### CODE REVIEW

File: _data/students/andrewfhou.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Andrew Hou 
+github: andrewfhou
+image: andrewfhou.png
+twitter: 3245355833
```
Comment (m-murad) [inline _data/students/andrewfhou.yml]: Enter a valid Twitter handler

**Comment by andrewfhou** [comment]
Added valid twitter handle (thought when it said ID, it meant the actual twitter ID)

File: _data/students/andrewfhou.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: Andrew Hou 
+github: andrewfhou
+image: andrewfhou.png
+twitter: @umbraah
```
Comment (opticod) [inline _data/students/andrewfhou.yml]: Remove `@`

**Comment by andrewfhou** [comment]
@opticod removed @

**Pull request merged**